README.md

```diff
@@ -9,9 +9,10 @@ A CircleCI Orb for Salus
 | salus_executor | CircleCI executor to use that specifies Salus environment | `coinbase/salus:2.4.1` | See [executor reference](https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21)|
```

Comment:
We should update this to reflect that we now use `coinbase/salus:latest` as default:
```suggestion
 | salus_executor | CircleCI executor to use that specifies Salus environment | `coinbase/salus:latest` | See [executor reference](https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21)|
```

Reply:
I think pegging to a known working version is preferred over the latest tag, this change wasn't intentional so I reverted it (and updated to 2.4.2).

Reply:
This sounds good to me. Is the intention that the latest production orb version follows the latest stable Salus version? If so, this means we have to follow versioning closely and potentially make releases following any version update. I'm not against this, as long as we're clear on what to expect as default. What do you think makes sense?

Reply:
Yeah, we'll have to follow versioning closely and release new ones when Salus does. I'm happy to help maintain this as I'll be closely following Salus' development, and if it becomes a burden we can shift to latest tag. 

When I said preferred, I was referencing this comment: https://github.com/coinbase/salus/pull/41#issuecomment-487819029